couchdbkit/client.py

```diff
@@ -954,7 +960,7 @@ def fetch(self):
                 pass
         self._dynamic_keys = []
 
-        self._result_cache = self.fetch_raw().json_body
+        self._result_cache = self.fetch_raw()
```

Comment:
Does this eventually go to `raw_view` which now returns a dict? (not very obvious from this that `fetch_view` changes from returning a request to returning a dict

Reply:
I don't follow - there's no ```fetch_view``` function in couchdbkit.

Reply:
I think I see what you mean - ```self._fetch``` is ```raw_view```, which returns a ```cloudant.View(dict)``` instead of a ```restkit.Response```.

Reply:
thanks that makes sense.